src/js/pre-need/components/ServicePeriodView.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { formatReviewDate } from '../../common/schemaform/helpers';
+import { serviceLabels } from '../utils/labels';
+
+export default function ServicePeriodView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.dateRange) {
+    from = formatReviewDate(formData.dateRange.from);
+    to = formatReviewDate(formData.dateRange.to);
+  }
+
+  return (
+    <div>
+      <strong>{serviceLabels[formData.serviceBranch]}</strong><br/>
+      {from} &mdash; {to}
```

Comment:
Is there is a reason for &mdash; rather than —, as we use both, and with varying spacing conventions.

Reply:
Good question, I don't remember. I copied this from another component, so it's at least following what we normally do for the service periods view field.